includes/Modules/Idea_Hub.php

```diff
@@ -195,6 +195,23 @@ public function register() {
 			 */
 			add_filter( 'post_class', $this->get_method_proxy( 'update_post_classes' ), 10, 3 );
 
+			add_filter(
+				'googlesitekit_tracking_allowed',
+				function( $tracking_allowed ) {
+					if ( true === $tracking_allowed ) {
+						return $tracking_allowed;
+					}
+
+					// If tracking is not allowed yet, check whether it is posts list or dashboard page.
+					$screen = get_current_screen();
```

Comment:
We need to ensure this function exists first because the above filter will be called on the frontend as well if the SK admin bar is enabled.

Reply:
Updated.